roles/core/common/tasks/apt/set_vars.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: check | Get architecture
+  command: dpkg --print-architecture
+  register: ubuntu_arch
+  changed_when: false
+
+- name: check | Set scale_architecture
```

Comment:
Should these two heading be "set |" or "check |"

Reply:
will change it to check to set.

Reply:
done